### PR TITLE
G11n java client javadoc format

### PR DESCRIPTION
**Pull request by tigershi**

### CODE REVIEW

File: src/main/java/com/vmware/vipclient/i18n/base/instances/TranslationMessage.java [hunk 1]
```diff
@@ -21,8 +21,8 @@
 /**
  * This class provide some APIs to get translation from VIP service in
  * string-based, component-based level.
- *
- * For string-based level APIs, some need the source string as parameter, some don't, instead the source string is put to the resource bundle.
+ * <pre>
+ * <p>For string-based level APIs, some need the source string as parameter, some don't, instead the source string is put to the resource bundle.
```
Comment (Xiaochao8) [inline src/main/java/com/vmware/vipclient/i18n/base/instances/TranslationMessage.java:25]: What does <p> mean?

Comment (tigershi) [reply src/main/java/com/vmware/vipclient/i18n/base/instances/TranslationMessage.java:25]: format java doc

Comment (Xiaochao8) [reply src/main/java/com/vmware/vipclient/i18n/base/instances/TranslationMessage.java:25]: Is `</p>` missed?

**Comment by linr211** [comment]
what's the JIRA  or github ticket? I don't understand why needs to include the java doc and source files into the jar build.

**Review by linr211 — APPROVED** [review]

**Pull request merged**